mani_skill/examples/motionplanning/panda_stick/run.py

```diff
@@ -0,0 +1,154 @@
+import argparse
+import os.path as osp
+
+import gymnasium as gym
+import numpy as np
+from mani_skill.examples.motionplanning.panda_stick.solutions import \
+    solveDrawTriangle
+from mani_skill.utils.wrappers.record import RecordEpisode
+from tqdm import tqdm
+
+MP_SOLUTIONS = {"DrawTriangle-v1": solveDrawTriangle}
+
+
+def parse_args(args=None):
```

Comment:
thanks for adding this file. 

we might want to consider having a more unified motion planning interface for all robots with some base classes and run scripts etc., this should be fine for now though!